trillian/integration/hammer.go

```diff
@@ -407,61 +416,94 @@ func (s *hammerState) getRoots(ctx context.Context) error {
 	return nil
 }
 
+func (s *hammerState) statString() string {
```

Comment:
The name `statString` is a bit redundant -- either just `stat` (the `string` is implied by the return type) or (preferably) `String` (so it satisfies the `fmt.Stringer` interface).